diff --git a/dist/h3.mjs b/dist/h3.mjs
index b57f25a25a8b62a48adbb0d3d767e6bb3c509aaa..383bd5985552d8a9458791a10bae5718b6b92763 100644
--- a/dist/h3.mjs
+++ b/dist/h3.mjs
@@ -1,3 +1,3 @@
-import { H3Error, H3Event, createError, deleteCookie, getCookie, getRequestURL, sanitizeStatusCode, setCookie } from "h3";
+import { createError, deleteCookie, getCookie, getRequestURL, sanitizeStatusCode, setCookie } from "h3";
 export * from "h3";
-export { H3Error, H3Event, createError, deleteCookie, getCookie, getRequestURL, sanitizeStatusCode, setCookie };
+export { createError, deleteCookie, getCookie, getRequestURL, sanitizeStatusCode, setCookie };
